geode-core/src/distributedTest/java/org/apache/geode/cache30/GlobalRegionCCEDUnitTest.java

```diff
@@ -163,78 +150,58 @@ public void testTombstones() {
    */
   @Test
   public void testTombstoneExpirationRace() {
-    VM vm0 = VM.getVM(0);
-    VM vm1 = VM.getVM(1);
 
     final String name = this.getUniqueName() + "-CC";
-    SerializableRunnable createRegion = new SerializableRunnable("Create Region") {
+    SerializableRunnable createRegion = new SerializableRunnable() {
       @Override
       public void run() {
         try {
           RegionFactory f = getCache().createRegionFactory(getRegionAttributes());
           CCRegion = (LocalRegion) f.create(name);
+
+          // noinspection OverwrittenKey
           CCRegion.put("cckey0", "ccvalue");
+          // noinspection OverwrittenKey
           CCRegion.put("cckey0", "ccvalue"); // version number will end up at 4
         } catch (CacheException ex) {
```

Comment:
Optional: Delete this `catch` block. `CacheException` is a `RuntimeException` so it doesn't even requires a `throws` clause.

Reply:
done